lessons/topics/02 - Functions and Prototype/README.md

```diff
@@ -0,0 +1,31 @@
+# Functions and prototypes
+
+In this topic we are going to talk about the different kind of functions, and explain little by litle how to structure our code by adding functions and prototypes, making our code more modular and recyclable
+
+## Topics:
+
+* Functions keywords and structures
+* returns
+* Function composition
+* Function arguments
+* Closures
+* Prototype methodology 
+
+
+## Previous Knowledge
+
+* variables Assignment
+* functions
+* What does the prototype keyword mean in javascript
+* fat arrow functions
+* Scope
+* Inheritance
```

Comment:
same for Inheritance, if we write content about prototypes, this lesson should cover it.